common/djangoapps/student/tests/test_credit.py

```diff
@@ -118,7 +118,7 @@ def test_purchased_credit(self):
 
         # Expect that the user's status is "pending"
         response = self._load_dashboard()
-        self.assertContains(response, "credit-request-pending-msg")
+        self.assertContains(response, "credit-request-not-started-msg")
```

Comment:
does the inline comment directly above need to be updated along with this change?